hack/dev-env.sh

```
@@ -316,6 +384,9 @@ case "${ACTION}" in
     init-ocm)                        init_ocm ;;
     join-clusters)                   join_clusters ;;
     setup-mesh)                      setup_mesh ;;
+    install-metallb)                 install_metallb ;;
+    install-gateway-api)             install_gateway_api ;;
+    clean)                           clean ;;
     *)
-        err "Unknown action: '${ACTION}'. Valid: check-host, create-cluster, install-olm, install-cert-manager, install-managed-serviceaccount, init-ocm, join-clusters, setup-mesh" ;;
+        err "Unknown action: '${ACTION}'. Valid: check-host, create-cluster, install-olm, install-cert-manager, install-managed-serviceaccount, init-ocm, join-clusters, setup-mesh, install-metallb, install-gateway-api, clean"" ;;
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)